server/mock/datastore_mock.go

```diff
@@ -1691,9 +1691,9 @@ type DataStore struct {
 
 	MDMWindowsGetEnrolledDeviceWithDeviceIDFunc        MDMWindowsGetEnrolledDeviceWithDeviceIDFunc
 	MDMWindowsGetEnrolledDeviceWithDeviceIDFuncInvoked bool
-	
+
```

Comment:
unrelated changes? or did we miss to generate the mock in another pr?

Reply:
Must have been missing from another PR. This was auto-generated when I ran the CI tests locally.